eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -1,10 +1,52 @@
 import React from "react";
-
+import { Customer, BookingsMeta, Slot } from "eisbuk-shared";
+import { fb2Luxon } from "@/utils/date";
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface Props {}
+type UserBooking = BookingsMeta & Pick<Customer, "certificateExpiration">;
+
+interface Props extends Slot<"id"> {
+  userBookings: UserBooking[];
+  absentees?: string[];
+}
+
+// mark attendees
+const AttendanceCard: React.FC<Props> = ({
+  date,
+  durations,
+  categories,
+  userBookings,
+  absentees,
+}) => {
+  // convert timestamp to luxon for easier processing
+  const luxonStart = fb2Luxon(date);
+
+  // convert durations to number values
+  const durationNumbers = durations.map((duration) => Number(duration));
+  const longestDuration = Math.max(...durationNumbers);
+
+  // get end time of longest duration (we're still using durations here, so it's pretty straightforward)
+  const luxonEnd = luxonStart.plus({ minutes: longestDuration });
 
-const AttendanceCard: React.FC<Props> = () => {
-  return <div />;
+  // get time for rendering
+  const timeString = `${luxonStart
+    .toISOTime()
+    .substring(0, 5)} - ${luxonEnd.toISOTime().substring(0, 5)}`;
```

Comment:
Better write this as:
```typescript
const startTime = luxonStart.toISOTime().substring(0, 5)
const endTime = luxonEnd.toISOTime().substring(0, 5)

const timeString = `${startTime} - ${endTime}`
```

This way it's readable straight up. Try and avoid multi-line string literals.